lib/ansible/modules/network/eos/eos_config.py

```diff
@@ -288,53 +311,158 @@ def main():
         replace=dict(default='line', choices=['line', 'block', 'config']),
 
         defaults=dict(type='bool', default=False),
-
         backup=dict(type='bool', default=False),
-        save=dict(default=False, type='bool'),
 
-        # deprecated arguments (Ansible 2.3)
-        config=dict(),
-        # this argument is deprecated in favor of setting match: none
-        # it will be removed in a future version
+        save_when=dict(choices=['always', 'never', 'changed'], default='never'),
+
+        diff_against=dict(choices=['startup', 'session', 'intended', 'running'], default='session'),
+        diff_ignore_lines=dict(type='list'),
+
+        running_config=dict(aliases=['config']),
+        intended_config=dict(),
+
+        # force argument deprecated in ans2.2
         force=dict(default=False, type='bool'),
```

Comment:
Not sure if you can set a message to tell them what to use, would be good to add.